templates/JSResolverOCHTTPS.js

```diff
@@ -1102,7 +1102,7 @@ function getReturnBlockFromSelection(selection, querySchemaInfo) {
 function resolveGrapgDBqueryForGraphQLMutation (queryAst, querySchemaInfo) {
 
     // createNode
-    if (querySchemaInfo.name.startsWith('createNode') && !querySchemaInfo.graphQuery) {
+    if (/^.*createNode/.test(querySchemaInfo.name) && !querySchemaInfo.graphQuery) {
```

Comment:
It would be better for performance to just use `querySchemaInfo.name.includes('createNode')` which is essentially the same check as that regex pattern.

```suggestion
    if (querySchemaInfo.name.includes('createNode') && !querySchemaInfo.graphQuery) {
```

Reply:
querySchemaInfo.name is createAirport for example. that would be createcreateAirport

Reply:
But I can change it to includes I think

Reply:
You can look at the other fields in `querySchemaInfo`, the node label should be there because it's used to generate the cypher query which requires the node label. Maybe `returnType`?